package.json

```diff
@@ -74,7 +74,6 @@
     "bs58": "^4.0.1",
     "buffer-peek-stream": "^1.0.1",
     "byteman": "^1.3.5",
-    "callbackify": "^1.1.0",
```

Comment:
`callbackify` is used here https://github.com/ipfs/js-ipfs/blob/813048ffb32a11cfefc51c2ec8634faaff2a924e/src/core/components/files-mfs.js so will need to be migrated to `promise-nodeify`. Would be rad if you also migrated `libp2p-kad-dht` also 😉🙏:

```
ipfs@0.37.0-rc.0 /Users/alan/Code/pl/ipfs/js-ipfs
├── callbackify@1.1.0 
└─┬ libp2p-kad-dht@0.15.3
  └── callbackify@1.1.0  deduped
```